listing-bundle/contao/templates/info/info_default.html5

```diff
@@ -1,5 +1,16 @@
+<?php
 
-<div class="<?= $this->class ?> listing block"<?= $this->cssID ?><?php if ($this->style): ?> style="<?= $this->style ?>"<?php endif; ?>>
+$this->wrapperAttributes = $this
+    ->attr()
+    ->addClass('listing')
+    ->mergeWith($this->wrapperAttributes)
+;
+
+$this->extend('block_searchable');
+
+?>
+
+<?php $this->block('content'); ?>
 
   <?php if ($this->headline): ?>
     <<?= $this->hl ?>><?= $this->headline ?></<?= $this->hl ?>>
```

Comment:
This looks like it would output the headline twice now, once inside the headline block in `block_searchable` and once in the content block here.

Reply:
@leofeyer this still needs to be adjusted

Reply:
Ah, sorry. It says "Outdated" above, so I figured it had already been changed. Fixed in bf5e8ee0ab800050324bee684a56ddd67f69337b.